ui/components/DataTable.tsx

```diff
@@ -75,31 +75,18 @@ const TableButton = styled(Button)`
 
 type Row = any;
 
-function defaultSortFunc(sort: Field): Sorter {
-  return (a: Row) => {
-    return a[sort.value as string];
-  };
-}
-
-export const sortWithType = (rows: Row[], sort: Field) => {
-  const sortFn = sort.sortValue || defaultSortFunc(sort);
-  return (rows?.slice() || []).sort((a: Row, b: Row) => {
-    switch (sort.sortType) {
-      case SortType.number:
-        return sortFn(a) - sortFn(b);
-
-      case SortType.date:
-        return Date.parse(sortFn(a)) - Date.parse(sortFn(b));
-
-      case SortType.bool:
```

Comment:
I think we can remove `SortType` from everywhere now too actually?

Reply:
actually yes we can do this